examples/ShowPlayer.cpp

```diff
@@ -126,6 +126,11 @@ int ShowPlayer::Playback(unsigned int iterations,
     }
   }
 
+  if (m_iteration_remaining > 0) {
+    OLA_WARN << "More iterations requested than can fit in the duration "
+                "(" << m_iteration_remaining << " iteration(s) remain)";
+  }
```

Comment:
Should this not go in verify instead?

Reply:
Given that other usage-type problems (e.g. stop point beyond show file end) are warned during playback, this seemed a fitting place for the warning.

Reply:
Hmm yeah, can I say both? :smile: It would be nice to spot config issues during validation, rather than needing to wait for a 30 minute show to run, but it would also be good to see them during show run...

Reply:
Because the verify function is the player without output, it appears in both places.  For instance, with a recorded show file about 11 seconds long:

```
ola_recorder --verify recorded.show --iterations 3 --duration 1
------------ Summary ----------
For all (3) iterations:
After playing for 1 second(s) total:
Universe 1: 4 frames
Universe 2: 40 frames

Total frames: 44
Total playback time: 1 seconds
examples/ShowPlayer.cpp:130: More iterations requested than can fit in the duration (3 iteration(s) remain)
```

Reply:
Ah, I forgot, and got confused by your comment about it being during playback!